src/manager.js

```diff
@@ -544,7 +554,6 @@ AdManager.prototype.refreshSlots = function(slotsToLoad, domElement) {
       changeCorrelator: false
     });
   } else {
-    window.headertag = window.googletag;
```

Comment:
@imakeinterfaces, we may want to move this into its own separate PR to get it out sooner as a patch to the lib while we do some more testing on the SRA stuff

Reply:
@spra85 that makes sense, keeping it here but will get that PR out once we get confirmation from index.